### ISSUE TITLE
Bad commit breaking existing apps

### ISSUE THREAD

**Issue by cloudeighto** [elpheria/rpc-websockets#92]
https://github.com/elpheria/rpc-websockets/commit/30a2192a7f6ae7fe3a0c014eedb354e2f1f51a8e

This commit is erroring for the solana blockchain's websockets with:

```
client.js:422 Uncaught TypeError: Cannot read property 'promise' of undefined
    at WebSocket.eval (client.js:422)
```
on json rpc messages that don't provide a `message.id`

Additionally, this commit adds spammy console logs.

@mkozjak

**Comment by cloudeighto** [comment]
<img width="470" alt="screen" src="https://user-images.githubusercontent.com/71376640/93314413-8f679a80-f7be-11ea-904a-4fa0202454ff.png">


screenshot of the params. I think its because the error field is undef?

**Comment by mkozjak** [comment]
@cloudeighto - Sorry for this. It should be fixed in 7.4.5 which is now released!
